### PR TITLE
FIX Some details in the documentation

### PR DESCRIPTION
**Pull request by dmoranj**
Fix some documentation and logging issues.

### CODE REVIEW

File: operations.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ A request has been received without a user Token, so it couldn't be identified.
 #### PROXY-GEN-003: Error initializing proxy: %s or Error initializing administration server: %s
 There was an error creating the HTTP server socket for the process. The specific nature of the error will be stated in the message. This error is critical, and makes the service unavailable.
 The most likely occurrence of this error will be when the IP address and port are already in use by other process (or an old instance of this service). Check the port is available using the netstat command.
+#### PROXY-GEN-003: Error initializing administration server: %s
```
Comment (fgalan) [inline operations.md]: Should it be PROXY-GEN-004? It seems to be a typo...

File: lib/errors.js [hunk 2]
```diff
@@ -108,6 +103,11 @@ var errors = {
         this.name = 'TARGET_SERVER_ERROR';
         this.message = 'There was an error redirecting the request to the target server: ' + msg;
         this.code = 501;
+    },
+    UserNotFound: function() {
+        this.name = 'USER_NOT_FOUND';
```
Comment (fgalan) [inline lib/errors.js]: Looking to documentation, I would have expected "USER_CREDENTIALS_NOT_FOUND". Is USER_NOT_FOUND intentional?

File: operations.md (hunk 1 above)
Comment (dmoranj) [inline operations.md]: It is. Fixing it.

File: lib/errors.js (hunk 2 above)
Comment (dmoranj) [inline lib/errors.js]: Another typo. Fixing...

Comment (dmoranj) [inline lib/errors.js]: Fixed in 15c57ab

File: operations.md (hunk 1 above)
Comment (dmoranj) [inline operations.md]: Fixed in 15c57ab

**Comment by fgalan** [comment]
LGTM

**Pull request merged**